plugins/kotlin/j2k/shared/src/org/jetbrains/kotlin/nj2k/tree/declarations.kt

```diff
@@ -281,5 +281,5 @@ class JKKtInitDeclaration(block: JKBlock) : JKInitDeclaration(block) {
 
 class JKJavaStaticInitDeclaration(block: JKBlock) : JKInitDeclaration(block) {
     override val isStatic: Boolean get() = true
-    override fun accept(visitor: JKVisitor) = visitor.visitJavaStaticInitDeclaration(this)
+    override fun accept(visitor: JKVisitor) = error("We shouldn't hit JKJavaStaticInitDeclaration during the printing phase")
```

Comment:
Since `JKVisitor` is only used in the printing phase (and I can't think of any other uses that might come up in the near future), I actually thought we could rename it to something like `JKPrintingVisitor`